README.md

```diff
@@ -589,24 +571,21 @@ Number of JWTs using keys:
 - ed24f836-5e22-4e0c-b9b2-dc4ee85128a9 : 3 ;
 ```
 
-You can see that our new entries got encrypted using the valid keys. Our application is therefore able to read fields encrypted with expired keys but new entries would be encrypted using the latest keys.
-This is a good news as this assure us that once we will have migrated the old field with the latest key, no application will produce rows encrypted with the old keys.
-
+You can see that the new entries are encrypted using the valid keys. Your application is therefore able to read fields encrypted with expired keys but new entries will always be encrypted using the latest keys. Note that the first two ids are the valid `kid` values for the valid keys and the last two are the `kid` values for the expired keys. This is good news as it proves that once you migrate fields encrypted with expired keys to the latest keys, the application will no longer contain any rows encrypted with the old keys. 
 
-### Step 3: Migrate our old fields by re-encrypting them with the new valid keys.
 
-If you remember in step 2, we voluntary didn't ran the `jose-reencrypt-database` docker container. This was a good way to show the importance of this migration.
+### Step 3: Migrate the old fields by re-encrypting them with the new valid keys.
+In step 2, you didn't run the `jose-reencrypt-database` docker container in order to see the importance of migration.
 
-Let's fix the state of our database by running the re-encryption batch job:
+You can now run the re-encryption batch job to migrate the database:
 
 ```
 docker-compose up jose-reencrypt-database
```

Comment:
`docker-compose up -d jose-reencrypt-database` - Means the user doesn't have to escape the terminal once the job is complete

Reply:
I was hesitating to do it but I think it's the right choice. It's easier at the end.